internal/cri/config/config.go

```diff
@@ -603,6 +610,67 @@ func ValidateServerConfig(ctx context.Context, c *ServerConfig) ([]deprecation.W
 	return warnings, nil
 }
 
+func (config *ContainerdConfig) loadRuntime(handler string) (Runtime, error) {
+	r, ok := config.Runtimes[handler]
+	if ok {
+		return r, nil
+	}
+
+	p := filepath.Join(defaults.DefaultConfigDir, "cri.runtimes", handler, "config.toml")
```

Comment:
- Needs docs
- Is there any hope that the directory can be potentially exposed to non-CRI applications? Then no need to prepend the "cri." prefix
- For the consistency with `certs.d`, maybe the directory should have the `.d` suffix
- `config.toml` can be easily confused with `/etc/containerd/config.toml`. Maybe `runtime.toml`?

Reply:
Possibly. Right now the config type lives in criconfig.
Need to take a look if it makes sense to pull it out.

Reply:
Actually not sure here given the use of pod_annotations and sandboxer.

Reply:
Also some cni specific stuff there as well.

Reply:
I've updated this to drop the `cri.`, however I have not made any changes to the cri config.
Everything other than changes specific to the config we use for this should be good to go.